int.env

```diff
@@ -0,0 +1,6 @@
+SPHINX_PORT=9312
+SPHINX_EFS=/var/local/efs-dev/geodata/service-sphinxsearch/int/index/
+PGPASS=www-data
+PGUSER=www-data
+PGHOST=master.chtopodb.bgdi.ch
+DBSTAGING=int
```

Comment:
dito, see above

Reply:
@ltshb @boecklic and how can we use them without the infra-vhost project / docker-compose file. 
p.e. the search index maintenance (index updates, etc)  will not be executed with the infra-vhost setup, this will be executed as ``DB=stopo_dev.public.ch make pg2sphinx`` on the data development host.

Reply:
@ltclm as I don't have another proposal for this use case it is fine for me to leave these infra settings in the repo, however as a best practice I would avoid setting credential in plain text here (even though those are not real one :-)) and use gopass with summon. It is also better for maintenance if we in future change this credential we only have to change it in gopass and don't have to touch each repo that use them.